apps/web/src/app/(dashboard)/documents/data-table.tsx

```diff
@@ -92,7 +92,13 @@ export const DocumentsDataTable = ({
           {
             header: 'Status',
             accessorKey: 'status',
-            cell: ({ row }) => <DocumentStatus status={row.getValue('status')} />,
+            cell: ({ row }) => {
+              const status = row.original.deletedAt
+                ? ExtendedDocumentStatus.BIN
+                : (row.getValue('status') as ExtendedDocumentStatus);
+
+              return <DocumentStatus status={status} />;
```

Comment:
`Bin` matches the UI text and implies items can be recovered, unlike `Deleted`. `In Bin` would have worked but then it will be the only tab with more than a single word the others are single.